src/Microsoft.DocAsCode.Build.Engine/PostbuildPhaseHandlerWithIncremental.cs

```diff
@@ -88,6 +92,28 @@ private void ReloadModelsPerChanges(IEnumerable<HostService> hostServices)
             }
         }
 
+        private void RegisterUnloadedXRefSpec()
+        {
+            var lastXrefMap = LastBuildVersionInfo?.XRefSpecMap;
+            foreach (var m in _unloadedFiles)
+            {
+                if (lastXrefMap == null)
+                {
+                    throw new BuildCacheException($"Full build hasn't loaded XRefMap.");
+                }
+                IEnumerable<XRefSpec> specs;
+                if (!lastXrefMap.TryGetValue(m, out specs))
```

Comment:
so unloaded files contain overwrite files? Are overwrite files also inside XrefMap?

---
In reply to: [90229696](https://github.com/dotnet/docfx/pull/936#discussion_r90229696) [](ancestors = 90229696)